.gitmodules

```diff
@@ -6,3 +6,7 @@
 	path = template
 	url = https://github.com/abloomston/workwithme.guide-template.git
 	branch = master
+[submodule "workwithme.guide-myguide"]
```

Comment:
👋 @craychee nice to meet you! I've just modified the instructions for submitting a WWM guide to be clearer in https://github.com/abloomston/workwithme.guide/pull/12, apologies for any confusion.  The way this submodule is added, your link would be https://workwithme.guide/workwithme.guide-myguide.

To change it so your link is e.g. https://workwithme.guide/craychee:

1. Remove this submodule (https://stackoverflow.com/questions/1260748/how-do-i-remove-a-submodule) to start fresh
2. ```git submodule add -b master https://github.com/craychee/workwithme.guide-myguide.git craychee```
3. stage and push changes to your `master`

Reply:
@abloomston 👋 Thank you for the instructions! I guess I was just mindlessly following instructions :) TIL about `deinit`.

I think I have reset properly. What a fun exercise.